bindata/network/ovn-kubernetes/controller.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: ovn-kubernetes
+  name: sdn-controller-config
+data:
+  controller-config.yaml: |-
+{{.NetworkControllerConfig | indent 4}}
```

Comment:
So this is a template for a ConfigMap. In the OpenShift SDN case, it gets filled in with the YAML config data that the OpenShift SDN controller needs, and then that data gets mounted as `/config/controller-config.yaml` in the SDN controller pod, and the pod spec below invokes `hypershift openshift-network-controller --config=/config/controller-config.yaml`.

You need to redo this in a way that works for ovn-kube. Since ovn-kube doesn't take a config file, you'll probably want to use the ConfigMap to fill in pod environment variables rather than a file (see the ConfigMap docs in kubernetes if you're not familiar with this), and define a bunch of single variables here like `K8S_APISERVER` and `CLUSTER_SUBNET` and such, and then in the pod spec, suck the ConfigMap into the environment and then use the environment variables in the `ovnkube` command line.

Oh, actually, ovnkube-master.yaml and ovnkube.yaml below are already written to be pulling data out of a configmap called `ovn-config`, so that's what you want to be creating a template for here.

Reply:
@danwinship ovnkube-master.yaml and ovnkube.yaml are copied from ovn-kubernetes and have not yet be edited. I am not sure how we will keep the daemonsets in sync between the operator and ovn-kubernetes.

Reply:
The details of the daemonsets presumably will not change much, so there will not need to be much "keeping in sync". And if there is, then there is. We need to have our own daemonset here, which is independent of the upstream one. It's a fork, not a copy.